test/codegen-generated-classes-test/src/test/resources/jetty-logging.properties

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#  http://aws.amazon.com/apache2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+#
+
+# Set up logging implementation
+org.eclipse.jetty.util.log.class=org.eclipse.jetty.util.log.StdErrLog
```

Comment:
What is jetty logging used for?

Reply:
We are using wiremocks as the local server and it's using jetty under the hood. This file is added to turn off the jetting logs so that it doesn't spam our logs.

Reply:
Ok, thanks for explaining!